src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepExecution.java

```diff
@@ -424,23 +423,6 @@ public String getCookie() {
         }
 
         @Override public CauseOfBlockage getCauseOfBlockage() {
-            if (FlowExecutionList.get().isResumptionComplete()) {
```

Comment:
I think I will preserve this API in `workflow-api` temporarily after I revert PRs over there just in case users update `workflow-api` before `workflow-durable-task-step`, but I will just make it always return `false`.